### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lauraperezco**
README.md refresh to match 2.0 wording and add images.

See issue https://github.com/meltano/internal-general/issues/337 for more information

### CODE REVIEW

**Review by aaronsteers — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -1,45 +1,50 @@
-# Meltano: Your DataOps OS
+![](https://lh6.googleusercontent.com/ZTyjEphqTxYMMfU52V6gA21HP1kHhLGFUPaO_eU7eQKfZwbqq2xsao86vlRJ_YPeTT7oJ4QOM7usLJ1q0fVYt8eDSMnqCaQpYjsS7cam5lgdQL-QQ4FxjGDSCKkp_B5LCl0cwEfOf0RjAldHYQ)
```
[the remark this answers is not in the document]
Comment (lauraperezco) [reply README.md:1]: @aaronsteers nope, they're just our logo and the soon-to-be new website animation. But I didn't mean to use tracking codes on them or anything

[the remark this answers is not in the document]
Comment (aaronsteers) [reply README.md:1]: Thanks, @lauraperezco. No need to block on this on change anything, but I wonder in the long run if we should prefer storing the images within the repo so they can be passed in the same webserver/webservice.

File: README.md [hunk 2]
```diff
@@ -1,45 +1,50 @@
...
 
-Meltano [embraces](https://handbook.meltano.com/product/singer) the [Singer](https://www.singer.io/) standard and its community-maintained library of open source
-[extractors](https://hub.meltano.com/extractors/) and
-[loaders](https://hub.meltano.com/loaders/),
-and leverages [dbt](https://www.getdbt.com) for [transformation](https://docs.meltano.com/guide/transformation).
+Integrations
+------------
 
-## Documentation
+[MeltanoHub](https://hub.meltano.com/) is the single source of truth to find any Meltano plugins as well as [Singer](https://singer.io/) taps and targets. Users are also able to add more plugins to the Hub and have them immediately discoverable and usable within Meltano. The Hub is lovingly curated by Meltano and the wider Singer community.
```
Comment (tayloramurphy) [inline README.md:14]:
```suggestion
[MeltanoHub](https://hub.meltano.com/) is the single source of truth to find any Meltano plugins as well as [Singer](https://singer.io/) taps and targets. Users are also able to add more plugins to the Hub and have them immediately discoverable and usable within Meltano. The Hub is lovingly curated by Meltano and the wider Meltano community.
```

File: README.md [hunk 3]
```diff
@@ -1,45 +1,50 @@
...
 
-Check out the ["Getting Started" guide](https://docs.meltano.com/getting-started)
-or find the full documentation at <https://docs.meltano.com>.
+Installation
+------------
 
-## Contributing to Meltano
+If you're ready to run an EL(T) pipeline with a data source and destination of your choosing, start by following the [Installation guide](https://docs.meltano.com/guide/installation) to have Meltano up and running in your device.
```
Comment (tayloramurphy) [inline README.md:19]: What about something like "If you're ready to build your ideal data platform and start running data workflows, start by..."

```suggestion
If you're ready to build your ideal data platform and start running data workflows across multiple tools, start by following the [Installation guide](https://docs.meltano.com/guide/installation) to have Meltano up and running in your device.
```

File: README.md [hunk 4]
```diff
@@ -1,45 +1,50 @@
...
+Meltano
+-------
 
-[Meltano](https://meltano.com) is
-[open source](https://gitlab.com/meltano/meltano),
-[self-hosted](https://docs.meltano.com/guide/production),
-[CLI-first](https://docs.meltano.com/reference/command-line-interface),
-[debuggable](https://docs.meltano.com/reference/command-line-interface#debugging), and
-[extensible](https://docs.meltano.com/concepts/plugins).
+Welcome to the open source DataOps Infrastructure to manage and centralize all the data tools in your stack and turn them into your ideal end-to-end data platform. With Meltano, you'll be able to collaboratively build and improve your data platform like a software project; spinning up a service or tool (Airflow, dbt, Great Expectations, Snowflake, etc) and easily configuring, deploying and managing it through a single control plane.
```
[the remark this answers is not in the document]
Comment (lauraperezco) [reply README.md:7]: @tayloramurphy in this context I think your proposal of "**Your** open source..." sounds very good. What do you think?

[the remark this answers is not in the document]
Comment (tayloramurphy) [reply README.md:7]: @lauraperezco agreed!

**Pull request merged**